addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -49,6 +49,19 @@ def _prepare_mail_values(self, res_ids):
                 )
                 if body:
                     mail_values['body_html'] = body
+            if mail_values.get('body'):
+                mail_values['body'] = Markup(
+                    '<div><span>{mailing_sent_message}</span></div>'
+                    '<blockquote class="border-start" data-o-mail-quote="1" data-o-mail-quote-node="1">'
+                    '{original_body}'
+                    '</blockquote>'
+                ).format(
+                    mailing_sent_message=_(
+                        'Received this mailing: %(mailing_name)s', 
```

Comment:
Isn't it "Sent this mailing" ? Well it depends on the point of view, but it seems we take the one from the sender, not the receiver :D . Or maybe just "Mailing 'name' sent" ?